pkg/network/protocols/http/monitor.go

```diff
@@ -35,12 +36,14 @@ var (
 // * Consuming HTTP transaction "events" that are sent from Kernel space;
 // * Aggregating and emitting metrics based on the received HTTP transactions;
 type Monitor struct {
-	consumer       *events.Consumer
+	httpConsumer   *events.Consumer
+	http2Consumer  *events.Consumer
 	ebpfProgram    *ebpfProgram
 	telemetry      *telemetry
 	statkeeper     *httpStatKeeper
 	processMonitor *monitor.ProcessMonitor
 
+	http2Enabled bool
```

Comment:
why?

Reply:
the http2 should be config in the monitor not?